backends/ebpf/targets/kernel_target.py

```diff
@@ -145,6 +154,44 @@ def _attach_filters(self, bridge, proc):
             return result
         return testutils.SUCCESS
 
+    # def _attach_filters(self, bridge: Bridge, proc: subprocess.Popen) -> int:
```

Comment:
Do you think this commented-out function is likely to be useful in the future?

Reply:
Ah, looks like this is the wrong branch... making this draft again. Losing track of all the PRs. Apologize for the premature ping!

Reply:
Turns out this code was commented out for compatibility reasons. It works but not for all distros and setups. I would keep it commented for now and fix it in a later PR. Right now, we should get this version in to make progress on the Ubuntu 24.04 support, which requires some of the changes here.